edweiss-firebase/functions/src/functions/lectures/createQuestion.ts

```diff
@@ -2,14 +2,14 @@ import { Timestamp } from 'firebase-admin/firestore';
 import LectureDisplay from 'model/lectures/lectureDoc';
 import { onAuthentifiedCall } from 'utils/firebase';
 import { addDocument, CollectionOf, Collections, getRequiredDocument } from 'utils/firestore';
-import { fail, ok } from 'utils/status';
+import { fail, NOT_IN_COURSE, ok, USER_NOT_FOUND } from 'utils/status';
 
 
 export const createQuestion = onAuthentifiedCall(LectureDisplay.Functions.createQuestion, async (userId, args) => {
-    const user = await getRequiredDocument(Collections.users, userId, fail('user_not_found'));
+    const user = await getRequiredDocument(Collections.users, userId, USER_NOT_FOUND);
 
     if (!user.courses.includes(args.courseId))
-        return fail('not_in_course');
+        return NOT_IN_COURSE;
```

Comment:
Oh I did not knew we could do this.
That's seems sick. I will try in the future to follow this insight.
Thanks for pointing it to myself.

Reply:
Yes but this just involves creating constants in utils/status, again I just followed Youssef's model for this